sdk/textanalytics/ai-text-analytics/test/public/expectations.ts

```diff
@@ -3835,49 +3819,29 @@ export const expectation26: AnalyzeBatchResult[] = [
       {
         id: "0",
         warnings: [],
-        sentiment: "positive",
-        confidenceScores: { positive: 0.99, neutral: 0.01, negative: 0 },
+        sentiment: "neutral",
+        confidenceScores: { positive: 0.04, neutral: 0.76, negative: 0.2 },
```

Comment:
NIT: In java and .net, we don't assert the confidence score anymore. Since the prediction model numerical score will be changed when the ML model is upgraded.

Reply:
Good point. I found an issue with confidence scores in this PR where the summation is not 1 and I included that in my bug report linked to above. I agree it can be annoying to update the scores every time the service pushes a change, but I don't mind so I can have a closer look at the changes.